_getting-started/index.md

```diff
@@ -4,21 +4,32 @@ title: Getting Started
 partof: getting-started
 languages: [ja]
 includeTOC: true
+scala3: true
```

Comment:
I think shouldn't be here. It causes the "Work in Progress" banner to appear, and we probably don't want that for this page.

Reply:
> Maybe we should show neither the Scala 2 nor the Scala 3 banner?

Ideally, yes, indeed. Not sure how easy that is to do given the current infrastructure.

Reply:
It turns out it’s not so simple because that banner not only shows “Scala 2” or “Scala 3”, but it also contains a navigation menu with several items (Learn, Reference, Glossary, etc.), and that navigation menu is different for Scala 2 and Scala 3.

So, I am tempted to go back to having two URLs (`/getting-started`, and `/scala3/getting-started`) showing the same content except the banner, which would be specialized.